tests/tests_pytorch/checkpointing/test_model_checkpoint.py

```diff
@@ -1171,6 +1171,59 @@ def test_ckpt_version_after_rerun_same_trainer(tmpdir):
     assert set(os.listdir(tmpdir)) == expected
 
 
+def test_ckpt_overriding_enabled_after_rerun_new_trainer(tmpdir):
+    """Check that previous checkpoints are renamed to have the correct version suffix when new trainer instances
+    are used."""
```

Comment:
I think that with the feature enabled, there aren't any suffixes? And there is no renaming. 
```suggestion
    """Check that previous checkpoints get overwritten an no suffixes are generated."""
```

Reply:
Missed to change description, my bad.